rest_framework_gis/filters.py

```diff
@@ -38,49 +38,68 @@
 
 
 __all__ = [
-    'InBBoxFilter',
-    'InBBOXFilter',
-    'GeometryFilter',
-    'GeoFilterSet',
-    'TMSTileFilter',
-    'DistanceToPointFilter',
-    'DistanceToPointOrderingFilter',
+    "InBBoxFilter",
+    "InBBOXFilter",
+    "GeometryFilter",
+    "GeoFilterSet",
+    "TMSTileFilter",
+    "DistanceToPointFilter",
+    "DistanceToPointOrderingFilter",
```

Comment:
please do not change the quoting style already in use, applies also to the other similar changes

Reply:
It is done by black. I will find all unnecessary modifications and revert them. 

At the end of all tests, while destroying the database, the `free` function throws an error. Please look into it. Not only mine but all PRs are failing because of this issue. 

```
Destroying test database for alias 'default'...
free(): invalid pointer
```